openshift/e2e-tests-openshift.sh

```diff
@@ -3,14 +3,17 @@
 
 source "$(dirname "$0")/e2e-common.sh"
 
-set +x
+set -Eeuox pipefail
+
+scale_up_workers || exit 1
 
 failed=0
 
 # Build binary
 (( !failed )) && build_knative_client || failed=1
 # Run unit tests
-(( !failed )) && run_unit_tests || failed=1
+# Temp disabled due to running into OOM
+# (( !failed )) && run_unit_tests || failed=1
```

Comment:
I have a suspicion that bumping memory too high in job's configuration leads to overcommitment on a container side during execution.

Reply:
A too high memory limit only has the danger that the pod can't be scheduled because of it. In fact the overcommitment effect is less the higher you set the resource limit. So OOM errors are probably caused by something different ?

Reply:
@dsimansk do you have a reference to the error for me ?

Reply:
https://prow.ci.openshift.org/view/gs/origin-ci-test/pr-logs/pull/openshift_knative-client/522/pull-ci-openshift-knative-client-release-next-45-e2e-aws-ocp-45/1330892816687042560#1:build-log.txt%3A1569

Reply:
With the new modular test setup, we could run unit test before the whole OCP create and run e2e test step. That'd give us a bit more controll for that container. TBH I'm not really sure what sorcery's happening inside the unit test exec :)

Reply:
FYI, the limit is already removed from the job. There's a test run here with enabled unit test: https://github.com/openshift/knative-client/pull/522 :crossed_fingers:

Reply:
Per debug PR it passed unit tests and failed on serving setup due to serving's `release-next` isn't synced with `master` yet. I was working fine with fork.

Reply:
IMO we should proceed with unit test in separate test step to don't mangle results together.